glib/src/param_spec.rs

```diff
@@ -501,6 +501,56 @@ macro_rules! define_builder {
                 self
             }
 
+            /// Mark the property as read only and drops the READWRITE flag set by default.
+            pub fn read_only(mut self) -> Self {
+                self.flags -= crate::ParamFlags::default();
```

Comment:
That seems like a bad idea. I think you want to remove only the `WRITABLE` flag here.

Or maybe work with a constant that has the flags you care about. But relying on the default value here seems like a bug waiting to happen.

Reply:
Right, I just assumed the default would always be readwrite